### ISSUE TITLE
Accessibility documentation update

### ISSUE THREAD

**Issue by Cr3aHal0** [primefaces/primeng#19400]
### Describe the bug

[Accessibility documentation](https://primeng.org/guides/accessibility) wrongly references `p-sr-only` css class despite `p-hidden-accessible` being the one used in the code and Prime* solutions.

### Pull Request Link

https://github.com/primefaces/primeng/pull/19399

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

.

### Environment

.

### Angular version

any

### PrimeNG version

v21

### Steps to reproduce the behavior

1. Goto accessibility doc
2. Read section about screen readers ([WAI-ARIA](https://primeng.org/guides/accessibility#wai-aria)) 

### Expected behavior

Doc should advise using `p-hidden-accessible` rather than `p-sr-only`
